src/Controller/ExportController.php

```diff
@@ -118,6 +118,7 @@ public function home(
 			'formats' => GeneratorSelector::getValidFormats(),
 			'format' => $this->getFormat( $request ),
 			'title' => $this->getTitle( $request ),
+			'wikiLangs' => $this->wikidata->getWikisourceLangs( $this->intuition->getLang() ),
```

Comment:
The `getWikisourceLangs()` method gives an array of lang codes to names and the `lang` twig var is always a code, so either this needs to be wrapped with `array_keys()`, or the check below needs to be something like `{% if wikiLangs[lang] is defined %}`.

Reply:
Or, we could use the `|keys` filter maybe like `{% if lang in wikiLangs|keys %}`?

Reply:
I added a commit that does just that.

Reply:
Yes, that's neater, good idea.